main.go

```diff
@@ -29,12 +29,14 @@ func main() {
 		session := session.Must(session.NewSession())
 		svc := ssm.New(session)
 
-		secrets, err := findSecrets(svc, ns)
-		if err != nil {
-			log.Fatal(err)
+		nsList := strings.Split(ns, ",")
```

Comment:
Could you please add a test case to verify 1) the new behavior works as expected and 2) this can handle malformed lists of namespaces?